src/sources/file/line_agg.rs

```diff
@@ -38,8 +46,9 @@ pub enum Mode {
     HaltWith,
 }
 
+/// Configuration parameters of the line aggregator.
 #[derive(Debug, Clone)]
-pub(super) struct Config {
+pub struct Config {
```

Comment:
Are these structs used beyond `super`? I'm wondering why the visibility change is necessary. Does this have to do with the pin projection?

Reply:
Cisibility changes are nit nesessary technically - it’s just a preparation to move this out of `file` mod.
I probably should do it right in this PR really.

Reply:
I'll do the move in another PR. There are pending changes to the `file` source - I'd like to merge those first to ease the conflicts. The plan is to move `src/sources/file/line_agg.rs` to `src/line_agg.rs` and `src/sources/file/mod.rs` back to `src/sources/file.rs`.